### PR TITLE
Add Noodles of Doom's Laundromat

### PR DESCRIPTION
**Pull request by jokermatt999**
All credit to Noodles of Doom. Uploaded since they don't 'gitgud' :)

### CODE REVIEW

**Comment by chaosvolt** [comment]
Yus, this should be fun. o3o

Additionally, for clarification I'll link back to the forum thread: http://smf.cataclysmdda.com/index.php?topic=10187.0

**Comment by Coolthulhu** [comment]
Something is wrong with the item spawns. Bleach either doesn't spawn at all or spawns in huge batches, easily 10 gallons on a single tile.

File: data/json/mapgen/laundromat.json [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      ["kariginu", 1],
+      ["kimono", 3]
+    ]
+},
+
+{
+      "type"  : "item_group",
+   "id"    : "laundromat_bleach",
+   "items" : [
+    ["bleach",        50]
```
Comment (chaosvolt) [inline data/json/mapgen/laundromat.json]: A lack of either a charges-min or count-min here is presumably the cause of massive amounts of bleach.

**Comment by Noodles-Of-Doom** [comment]
Sorry for the goof-up, folk ups :). I think I fixed the problem:

Changing the "chance" and "repeat" values of line 253...
{ "item": "laundromat_bleach", "chance": 75, "x": [6,12], "y": [19,20], "repeat": [1,2] }, 

...to this seems to fix the problem. Spawns are now more like 1-2 gallons of bleach on about 1-2 racks.
{ "item": "laundromat_bleach", "chance": 30, "x": [6,12], "y": [19,20], "repeat": [4,8] },

Also, thanks Jokermatt999 for PR-ing this- I'm not very git-savvy to say the least, so have my thanks for going out and putting this on Github :)

**Comment by Coolthulhu** [comment]
One thing more: it should also be less common. At weight 2, cities can easily have 4 'mats with barely any distance between them.

**Comment by jokermatt999** [comment]
Made those requested changes. Let me know if issues persist?

**Comment by Coolthulhu** [comment]
Good timing - I just started my testing run

**Comment by chaosvolt** [comment]
Ooh, will be interesting to see.

**Pull request merged**